### ISSUE TITLE
Support month locale

### ISSUE THREAD

**Issue by shifenis** [rinukkusu/simple_moment#18]
Like the title, it would be awesome if the library can support locale month.  
I mean, now it's possible to get the locale but unable to set month local. If set `Moment.setLocaleGlobally(locale);` and then try to get the month with ([see Moment Docs](https://momentjs.com/docs/#/parsing/special-formats/)) `Moment.now().format('LLLL')` you will always get the month in EN locale.

Am I wrong something or for now this feature doesn't exist?

**Comment by rinukkusu** [comment]
Hi @shifenis, I've looked into the matter and I had to extend some functionality to make it work. Because we're using the `intl` package as the backbone for formatting, you'd also need to initialize the chosen locale for `intl` to work with it.

## Option 1
Could the following be an option for you:

```dart
// initialize intl
var locale = LocaleDe();
await initializeDateFormatting(locale.localeString);

// set german locale
Moment.setLocaleGlobally(locale);

// format and print the current month in german
var formattedString = Moment.now().formatLocalized("LLLL"); // <-- new method
print(formattedString);
```

Here I went for a new method `formatLocalized` to not break existing functionality. 

## Option 2
But we could also set like a flag when setting the locale or something to tell the current `Moment` instance to use correct localization:

```dart
// initialize intl
var locale = LocaleDe();
await initializeDateFormatting(locale.localeString);

// set german locale
Moment.setLocaleGlobally(locale, useInFormat: true); // <--  enable usage in format

// format and print the current month in german
var formattedString = Moment.now().format("LLLL"); // <-- use regular format call
print(formattedString);
```

What do you think?

**Comment by shifenis** [comment]
Hi @rinukkusu, thanks for your quickly reply. 
Probably the second option looks better than the first one, cause it is more consistent: you don't have to use another method, just specify if you want to use the locale even for formatting.

**Comment by rinukkusu** [comment]
Released in `1.0.6` :)
